Assets/Scripts/Strategy/BaseManagement/HUD/IBuildingHUD.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class IBuildingHUD : MonoBehaviour
```

Comment:
Is this a class or an interface? And what is its purpose?

Reply:
c684ac7